marray/__init__.py

```diff
@@ -114,49 +115,95 @@ def mT(self):
     ## Methods ##
 
     # Methods that return the result of a unary operation as an array
-    unary_names = (['__abs__', '__floordiv__', '__invert__', '__neg__', '__pos__']
-                   + ['__ceil__'])
+    unary_names = ["__abs__", "__floordiv__", "__invert__", "__neg__", "__pos__"] + ["__ceil__"]
     for name in unary_names:
+
         def fun(self, name=name):
             data = self.call_super_method(name)
             return MaskedArray(data, self.mask)
+
         setattr(MaskedArray, name, fun)
 
     # Methods that return the result of a unary operation as a Python scalar
-    unary_names_py = ['__bool__', '__complex__', '__float__', '__index__', '__int__']
+    unary_names_py = ["__bool__", "__complex__", "__float__", "__index__", "__int__"]
     for name in unary_names_py:
+
         def fun(self, name=name):
             return self.call_super_method(name)
+
         setattr(MaskedArray, name, fun)
 
     # Methods that return the result of an elementwise binary operation
-    binary_names = ['__add__', '__sub__', '__and__', '__eq__', '__ge__', '__gt__',
-                    '__le__', '__lshift__', '__lt__', '__mod__', '__mul__', '__ne__',
-                    '__or__', '__pow__', '__rshift__', '__sub__', '__truediv__',
-                    '__xor__'] + ['__divmod__', '__floordiv__']
+    binary_names = [
+        "__add__",
```

Comment:
This is one of the things I am not a fan of. Perhaps if there is a rule that can be turned off, I can live with it.